stdlib/public/SwiftShims/RuntimeShims.h

```diff
@@ -59,6 +59,13 @@ int _swift_stdlib_putc_stderr(int C);
 SWIFT_RUNTIME_STDLIB_API
 __swift_size_t _swift_stdlib_getHardwareConcurrency(void);
 
+SWIFT_RUNTIME_STDLIB_API
```

Comment:
cc: @compnerd Is there a way where I can mark this `_INTERNAL` and allow the `LowLevel` module to be able to see it? I've been messing with cmake settings to see internal shims, but I didn't know exactly if it were possible.

Reply:
No, if it is internal, it means that only `swiftCore.so`/`swiftCore.dylib`/`swiftCore.dll` can see it